src/sage/rings/polynomial/differential_polynomial_ring.py

```diff
@@ -0,0 +1,44 @@
+from sage.rings.polynomial.ore_polynomial_ring import OrePolynomialRing
```

Comment:
Add general description of this module, here, together with licence and authors.